extensions/wikia/Email/i18n/ForgotPassword.i18n.php

```diff
@@ -0,0 +1,20 @@
+<?php
+$messages = array();
+
+$messages['en'] = array(
+	'emailext-password-subject' => 'Reset your Wikia password',
+	'emailext-password-summary' => 'It looks like you’re having trouble logging in. Sorry about that! As much as we’d like to just send you your password, for security reasons, we’ll need you to login with a temporary password.',
+	'emailext-password-intro' => 'Please see your temporary password below:',
+	'emailext-password-unrequested' => "If you didn't request a new password, don't worry! Your account is safe and secure. You can ignore this email and log in to Wikia with your old password.",
+	'emailext-password-questions' => 'Questions or concerns? Feel free to [http://community.wikia.com/wiki/Special:Contact/account-issue contact us].',
+	'emailext-password-signature' => 'The Wikia Community Team',
+);
+
+$messages['qqq'] = array(
+	'emailext-password-subject' => 'Subject of the forgot password email',
+	'emailext-password-summary' => 'Summary text for the forgot password email',
+	'emailext-password-intro' => 'Text shown just above the new temporary password',
+	'emailext-password-unrequested' => "Instructions for what to do if you didn't request a new password",
+	'emailext-password-questions' => 'Text on how to contact support.  The link here should be updated to point at the Wikia contact page for the appropriate language',
```

Comment:
Oh, I see you're calling out that i18n'd community URL here. If there was a way to have the code construct the URL I think that would be preferred, but if not this works fine too.